tests/schema_registry/conftest.py

```diff
@@ -164,8 +169,28 @@ def __init__(self, conf):
         adapter.register_uri('POST', self.subject_versions,
                              json=self.post_subject_version_callback)
 
+        adapter.add_matcher(self._auth_matcher)
         self._rest_client.session.mount('http://', adapter)
 
+    @classmethod
+    def _auth_matcher(cls, request):
+        headers = request._request.headers
+
+        # Pass request to downstream matchers
+        if 'Authorization' not in headers.keys():
```

Comment:
I'm not sure how this works under the hood, but this seems like it needs to scan keys() for 'Authorization', would suggest using `..keys().get('Authorization', None)`